### PR TITLE
Chore: update 'no swrs report found' legal text

### PR DESCRIPTION
**Pull request by dleard**

### CODE REVIEW

**Review by pbastia — APPROVED** [review]

File: app/containers/Applications/ApplyButtonContainer.tsx [hunk 1]
```diff
@@ -96,35 +96,37 @@ export const ApplyButton: React.FunctionComponent<Props> = ({
...
+              <em>Greenhouse Gas Industrial Reporting and Control Act</em>
+            </a>{' '}
+            or its regulations.
           </p>
           <p>
             For any questions, please <a href={adminMailToUrl}>contact</a> the
             Climate Action Secretariat at {adminEmail}
           </p>
         </Modal.Body>
+
```
Comment (kriscooke) [inline app/containers/Applications/ApplyButtonContainer.tsx:129]: From the document we were given, it appears this last paragraph about contacting us was removed

Comment (kriscooke) [reply app/containers/Applications/ApplyButtonContainer.tsx:129]: I don't see any note about the contact paragraph... We should probably try to clarify this first

Comment (kriscooke) [reply app/containers/Applications/ApplyButtonContainer.tsx:129]: I see it now - sorry! Looks like we're good to keep it.

**Comment by kriscooke** [comment]
@dleard Want to rebase and merge this one next?

**Review by kriscooke — APPROVED** [review]

**Pull request merged**